.github/workflows/newman.yml

```diff
@@ -0,0 +1,80 @@
+name: Postman Tests
+
+on:
+  push:
+    branches:
+      - main  # Change this to the appropriate branch if needed
+
+jobs:
+  postman_tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: centreon/tests/rest_api
+
+    env:
+      CONTAINER_NAME: my_centreon_container
+      CENTREON_URL: http://localhost
+      CENTREON_IMAGE: docker.centreon.com/centreon/centreon-web-alma9:develop
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Login to registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ secrets.DOCKER_REGISTRY_URL }}
+          username: ${{ secrets.DOCKER_REGISTRY_ID }}
+          password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+
+      - name: Install pnpm
+        run: npm install pnpm -g
+
+      - name: Install Dependencies
+        run: pnpm install -f ./centreon/tests/rest_api/package.json

```

Comment:
use `--frozen-lockfile` option

Reply:
why do you need to specify the path if you use the working-directory parameter?